ansible/condor/install.yml

```diff
@@ -39,9 +39,32 @@
   hosts: condor_submit
   roles:
   - condor-submit
+  - de-road-runner
 
 - name: add configuration for the condor worker nodes
   become: true
   hosts: condor_worker
   roles:
   - condor-worker
+  - de-docker-logging-plugin
+  - de-image-janitor
+  - de-network-pruner
+
+- name: log into required docker registries
+  hosts: condor_worker
+  become: true
+  tasks:
+    - community.docker.docker_login:
+        registry_url: "{{item.host}}"
+        username: "{{item.user}}"
+        password: "{{item.password | quote}}"
+        config_path: "~condor/.docker/config.json"
+      loop: "{{docker_registries}}"
+      when: docker_registries is defined
+
+    - ansible.builtin.file:
+        path: "~condor/.docker"
+        owner: "condor"
+        group: "condor"
+        recurse: true
+        state: "directory"
```

Comment:
I was getting errors when I tried to set `become_user` to true before logging in, so I simply specified the path to the configuration file then fixed the permissions after.